dotCMS/src/main/java/com/dotcms/contenttype/model/field/BinaryField.java

```diff
@@ -1,13 +1,19 @@
 package com.dotcms.contenttype.model.field;
 
+import java.util.Collection;
 import java.util.List;
 
+import com.dotmarketing.util.UtilMethods;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.util.UtilMethods'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)